### ISSUE TITLE
Slim PHP 4

### ISSUE THREAD

**Issue by abmcr** [harmony-betta/harmony#7]
Do you have planning an update to slim4 core? Thank you

**Comment by darkterminal** [comment]
Yes.. but I don't have so much time shortly. If you have an idea and you want to upgrade into it, just do some PR. Btw, thanks for asking.

Regards
